x-pack/plugins/apm/public/components/app/Correlations/index.tsx

```diff
@@ -41,32 +42,52 @@ export function Correlations() {
           setIsFlyoutVisible(true);
         }}
       >
-        View correlations
+        View significant terms
       </EuiButton>
 
```

Comment:
A minor visual improvement here would be to add a `<EuiSpacer size="s" />` to separate the button from the filters section.